client/rest-high-level/src/main/java/org/elasticsearch/client/RequestConverters.java

```diff
@@ -1174,6 +1175,17 @@ static Request xpackUsage(XPackUsageRequest usageRequest) {
         return request;
     }
 
+    static Request deleteLifecycle(DeleteLifecyclePolicyRequest deleteLifecyclePolicyRequest) {
```

Comment:
Rename to `deleteLifecyclePolicy`?

Reply:
Yep, +1 to rename this `deleteLifecyclePolicy`